src/app/modules/submission/submission.service.ts

```diff
@@ -385,11 +385,27 @@ export const downloadCleanFile = (cleanFileKey: string, versionId: string) => {
     })
     .createReadStream()
 
+  const timeDownloadStart = Date.now()
```

Comment:
/nit usually we have the convention of the datatype to be at the end of the variable i.e., `studentNricMap` over `mapStudentNric`.